src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const carCostPerDay = 40;
+  const mediumRentTermDiscount = 20;
+  const longRentTermDiscount = 50;
+  const price = carCostPerDay * days;
+
+  if (days < 3) {
+    return price;
+  }
+
+  if (days >= 3 && days <= 6) {
+    return price - mediumRentTermDiscount;
+  }
+
+  if (days >= 7) {
+    return price - longRentTermDiscount;
+  }
```

Comment:
Prefer using 'const' over 'let' where possible. In this case, 'price' is never reassigned, so it should be declared with 'const'.